### PR TITLE
Replaced all the css into tailwind css on SuccessStory component

### PR DESCRIPTION
**Pull request by subhajit20**
# Description of changes

Replaced all the css into tailwind css on SuccessStory component
Issue - #1725 

Before:
Before

After:
After

# Issue Resolved

Fixes #NA

### CODE REVIEW

File: components/SuccessStory/SuccessStory.js [hunk 1]
```diff
@@ -10,8 +9,8 @@ SuccessStory.propTypes = {
 
 function SuccessStory({ imageSource, quote, title }) {
   return (
-    <div className={styles.SuccessStory}>
-      <div className={styles.imageWrapper}>
+    <div className="box-border flex flex-col flex-nowrap items-center my-12 mx-4 pt-14 relative max-w-[320px] h-[940px] md:h-[748px] sm:h-auto">
+      <div className="absolute top-0 h-48  shadow-[1px_2px_5p_3px_rgba(0, 0, 0, 0.35)]">
```
Comment (kylemh) [inline components/SuccessStory/SuccessStory.js:13]:
```suggestion
      <div className="absolute top-0 h-48 shadow-[1px_2px_5p_3px_rgba(0, 0, 0, 0.35)]">
```

File: components/SuccessStory/SuccessStory.js [hunk 2]
```diff
@@ -21,8 +20,8 @@ function SuccessStory({ imageSource, quote, title }) {
         />
       </div>
 
-      <div className={styles.quote}>
-        <h6 className={styles.title}>{title}</h6>
+      <div className="flex flex-1 flex-col flex-nowrap items-center p-8  pt-40 border-2 border-solid border-primary [&blockquote]:my-4 [&blockquote]:mx-0 [&blockquote]:text-base">
```
Comment (kylemh) [inline components/SuccessStory/SuccessStory.js:23]:
```suggestion
      <div className="flex flex-1 flex-col flex-nowrap items-center p-8 pt-40 border-2 border-solid border-primary [&>blockquote]:my-4 [&>blockquote]:mx-0 [&>blockquote]:text-base">
```

File: components/SuccessStory/SuccessStory.js (hunk 1 above)
Comment (subhajit20) [reply components/SuccessStory/SuccessStory.js]: done

File: components/SuccessStory/SuccessStory.js (hunk 2 above)
Comment (subhajit20) [reply components/SuccessStory/SuccessStory.js]: done

**Pull request merged**